android/src/main/java/io/mosip/tuvali/openid4vpble/events/withArgs/VerificationStatusEvent.kt

```diff
@@ -0,0 +1,16 @@
+package io.mosip.tuvali.openid4vpble.events.withArgs
+
+import io.mosip.tuvali.openid4vpble.events.EventWithArgs
+
+class VerificationStatusEvent(private val status: VerificationStatus): EventWithArgs {
+  override fun getData(): HashMap<String, String> {
+    return hashMapOf("status" to status.value)
+  }
+
+  override val type = "onVerificationStatusReceived"
```

Comment:
[Minor one] For better readability and consistency, follow order of declarations like declare `type` first and then `getData`  or the other way around for all events

Reply:
sure